woocommerce/woocommerce-ios
# Adding a package UI - custom package scaffolding UI

Part of #13551


## Description


Basic UI scaffolding for the "Add package" screen and content for "Custom" package tab.

Rest of the functionalities will be done in **separate** stacked PRs (done on top of this branch or trunk if this gets merged before).

## Testing information


1. Install and set up the Woo Shipping extension on your store.
2. Build and run the app with the `revampedShippingLabelCreation` feature flag enabled.
3. Create an order with the processing status and at least one physical product.
4. In the order details, select "Create Shipping Label."
5. Tap "Select a Package" button
6. Tap on the package type button to change the type
7. Tap in length, width and height fields to input values and check that the "Add Package" button is enabled when all 3 values are entered
8. Toggle "save this as a new package template" ON and "Save package template" button will show.

## Screenshots


https://github.com/user-attachments/assets/30b88697-270d-4960-b956-4564fc6a8210

![356844982-de39b1f1-9e55-4d11-ac87-f3dce083f7fd-2](https://github.com/user-attachments/assets/ad306ed2-f904-4ba3-8118-f4f6c1bbb959)

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
